service/src/main/java/com/epam/esm/service/config/ServiceConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.esm.service.config;
+
+import com.epam.esm.dao.config.DBConfig;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import javax.sql.DataSource;
+
+@Configuration
+@Import({DBConfig.class})
+@EnableWebMvc
+@EnableTransactionManagement
+public class ServiceConfig {

```

Comment:
I wondering how service configuration knows where to find declared beans for service layer?

Reply:
rewrote configurations. components scan added